### DISCUSSION TITLE
Meeting 7 Agenda

### DISCUSSION THREAD

**Discussion by volcaniqueo** [bounswe/bounswe2023group1#84] [Meeting Agendas]
Let's add the entries for possible discussion topics for our upcoming meeting at 22 March.

**Comment by volcaniqueo** [comment]
Verification mechanism of the system, specifically the mechanism that the facilitator type is subject to.

**Comment by GulbeycanCagri** [comment]
I think we should also start to specify which information is required for different actions. We can also specify the information users can add to their profiles.

**Comment by kubraaksux** [comment]
label revision and usage - because we have many
Victim scope - controversion here
revision of mock-ups and scenarios
